Lib/enum.py

```diff
@@ -748,7 +751,10 @@ def __contains__(cls, value):
         try:
             return value in cls._value2member_map_
         except TypeError:
-            return value in cls._unhashable_values_
+            return (
+                    value in cls._unhashable_values_    # both structures are lists
```

Comment:
`# both structures are lists`
Is this comment redundant? It seems that the plural modifier in values ​​already represents a list.

Reply:
The plural would also work for dicts, tuples, etc., and the check would fail if value is unhashable when `cls._hashable_values` is a dict.